torch/_inductor/codegen/wrapper.py

```diff
@@ -786,17 +785,12 @@ def return_end_str(self):
 
     def generate_return(self, output_refs):
         if output_refs:
-            if len(output_refs) == 1:
-                self.wrapper_call.writeline(
-                    f"return {output_refs[0]};{self.return_end_str()}"
-                )
-            else:
-                self.wrapper_call.writeline(
-                    "return std::vector<at::Tensor>({"
-                    + ", ".join(output_refs)
-                    + "});"
-                    + self.return_end_str()
-                )
+            self.wrapper_call.writeline(
+                "return std::vector<at::Tensor>({"
+                + ", ".join(output_refs)
+                + "});"
+                + self.return_end_str()
+            )
```

Comment:
Python wrapper always returns a tuple regardless of whether the return value is a single Tensor or multiple tensors. Hence, the CppWrapper should always return `std::vector<at::Tensor>` to align the semantics.